ci/gpu/build.sh

```diff
@@ -42,8 +42,10 @@ logger "Check GPU usage..."
 nvidia-smi
 
 logger "Activate conda env..."
+# conda env create --name cuspatial_dev --file conda/environments/cuspatial_dev.yml
+# conda activate cuspatial_dev
 source activate gdf
-conda install "cudf=${MINOR_VERSION}.*" "cudatoolkit=$CUDA_REL" "gdal=2.4.*"
+conda install -c rapidsai-nightly -c default "cudf=0.12.*" "cudatoolkit=$CUDA_REL" "gdal=2.4.*"
```

Comment:
These channels should already be set in the container, should also continue to be able to use the minor version variable for cudf